tests/cases/compiler/largePrimitiveUnionComparisons.ts

```diff
@@ -0,0 +1,106 @@
+type PrimitiveUnion =
```

Comment:
You could simplify this to

```ts
type PrimitiveUnion =
    `x${0 | 1 | 2 | 3 | 4}${0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9}${0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9}${0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9}`;
```

Reply:
I was worried that, if I declared it as something other than an expanded union, we might change the representation in the future, silently invalidating the test.  For example, one can imagine specializing type comparison for string template types so that the union didn't have to be expanded.  TBH, string template types hadn't occurred to me, but I did specifically avoid using conditional spreads.

Reply:
I tried it and learned that the baseline captures the expansion, so we'll know if it changes.